src/eba_db.ml

```diff
@@ -117,7 +120,7 @@ let activation_table :
     Sql.view =
   <:table< activation (
        activationkey text NOT NULL,
-       userid bigint NOT NULL,
+       email text NOT NULL,
```

Comment:
The activationkey is for now the primary key of that table. 
I don't see for now, a lot of benefits at putting an id, what do you think ?